src/main/resources/templates/error/403.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<!-- Dodana przestrzeń nazw th -->
+<!-- Dodana przestrzeń layout -->
+
+<html lang="pl"
+      xmlns:th="https://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<head>
+    <meta charset="UTF-8">
+    <title>Strona nie istnieje - Psychoterapia Ewa Górska</title>
```

Comment:
title for page 403 has title from 404

Reply:
we can create common error template for pages 403 and 404. 

h2 and title - i think may have a variable with value from controller: "Brak dostępu"/ "Strona nie istnieje"